build/pre-migration-test-docker.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -ex
+
+echo "Preparing environment for fhir-server integration tests..."
+if [[ -z "${WORKSPACE}" ]]; then
+    echo "ERROR: WORKSPACE environment variable not set!"
+    exit 2
+fi
+
+# Set up installers and config files where docker processing can see them
+cd ${WORKSPACE}/fhir-install/docker
+./copy-dependencies-db2-migration.sh
+./copy-test-operations.sh
+
+# Stand up a docker container running the fhir server configured for integration tests
+echo "Bringing down any fhir server containers that might already be running as a precaution"
+docker-compose kill
+docker-compose rm -f
+
+echo "Bringing up db2... be patient, this will take a minute"
+docker-compose build --pull db2
+docker-compose up -d db2
+echo ">>> Current time: " $(date)
+
+# TODO wait for it to be healthy instead of just Sleeping
+(docker-compose logs --timestamps --follow db2 & P=$! && sleep 100 && kill $P)
+
+
+echo "Deploying the Db2 schema..."
+./deploySchemaAndTenant.sh
+
+echo "Migrating the schema to the latest version..."
+./copy-dependencies-db2.sh
```

Comment:
this will overwrite the "previous version" of the schema cli jar with the newly built one